python/README.md

```diff
@@ -68,15 +68,39 @@ randomly-generated 160-bit secret key.
 
 ## Development
 
-## Building the C++ extension module
+### Getting started  
 
-Mesh generation for segmentation volumes depends on a C++ extension module.  To
-build it, activate a suitable Python virtual environment and run:
+Some of the objects in this directory require mesh generation for the
+segmentation volumes, and that depends on a C++ extension module. To build it,
+you will need `gcc >= 5`. To see if your compiler is up to date, you can check
+in Terminal with:
+
+```shell
+gcc -v
+```
+
+If you need to update to a more recent version of gcc, you can follow the
+instructions in [this guide](https://gist.github.com/application2000/73fd6f4bf1be6600a2cf9f56315a2d91).
+
+You will also need an authorization token for both Google Tasks and AWS Simple
+Tasks. The easiest way is to copy them from another workstation in lab to your
+local machine so that these two files exist in these locations:
+
+```shell
```

Comment:
I also have created the boss-secret.json, but so far that's only relevant to the BossTransferTask which is used for pipeline imports.